src/index.tsx

```diff
@@ -31,6 +32,7 @@ ReactDOM.render(
           <Route path="/" exact={true} component={HomePageContainer} />
           <Route path="/issue/:id" exact={true} component={CallPageContainer} />
           <Route path="/done/:id" exact={true} component={DonePageContainer} />
+          <Route path="/impact" exact={true} component={MyImpactPageContainer} />
```

Comment:
The '/impact' link needs to be shown only when there is impact data for the user.

Reply:
This was confusing to users (including you!) so I think it may be more straightforward to provide some "no calls made yet" text and just have the impact link there all the time.